site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,282 @@
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Assurance that the components have been tested amongst the accessibility standard from the industry</p>
+            </li>
+            <li class="d-flex align-items-start mb-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Coverage of iOS and Android with app and SDK being available in open-source</p>
+            </li>
+          </ul>
+        </div>
+        <div class="col-12 col-md-6 offset-md-1 border-dark border border-1">
+          <iframe class="w-100 h-100" src="https://mastermedia.orange.com/pmJLYXq6kC" title="Explanation on how to get the app on your phone" allowfullscreen></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="py-4 py-md-5">
+    <div class="container-xxl py-2">
+      <h2 class="h1 mb-4 mb-md-5 text-primary text-center">How to download the showcase app</h2>
+      <div class="row pt-2 justify-content-around text-center">
+        <div class="col-12 col-sm-8 col-md-4 d-flex flex-column align-items-center">
+          <svg xmlns="http://www.w3.org/2000/svg" class="mb-4" width="6.25rem" height="6.25rem" role="img" focusable="false" aria-hidden="true">
+            <use xlink:href="#apple"/>
+          </svg>
+          <p class="h4 pt-3 mb-4">For iOS, simply hit the button below to download the app on your device.</p>
+          <div class="steps" id="ios-steps">
+            <a href="https://itunes.apple.com/fr/app/id6446178285" class="d-inline-block mt-3"><img loading="lazy" class="store-badge" src="/docs/{{< param docs_version >}}/assets/img/examples/download-app/app-store-badge.svg" alt="Get it on App Store" height="60"></a>
+          </div>
+        </div>
+        <hr class="d-md-none my-5 border-top border-1 border-translucent">
+        <div class="col-12 col-sm-8 col-md-4 d-flex flex-column align-items-center">
+          <svg xmlns="http://www.w3.org/2000/svg" class="mb-4" width="6.25rem" height="6.25rem" role="img" focusable="false" aria-hidden="true">
+            <use xlink:href="#android"/>
+          </svg>
+          <p class="h4 pt-3 mb-4">For Android, please make sure to follow the two steps process below.</p>
+          <div class="steps" id="android-steps">
+            <div>
+              <span class="tag my-3 border-primary">1</span>
+              <p class="fw-bold mb-3">Register with your Google store account to our <a href="https://groups.google.com/g/odsapp">Beta program list</a></p>
+              <a href="https://groups.google.com/g/odsapp" class="btn btn-secondary mb-3">Register</a>
+            </div>
+            <div class="vr mx-auto my-4 bg-body-secondary" style="height: 100px;"></div>
```

Comment:
Is it possible to make it in CSS?
Or at least to add `aria-hidden="true"` on it?

Reply:
It's feasible apart for the animation part I think.
Furthermore, I'm not quite sure that we need the `aria-hidden` attribute in here 🤔 Otherwise we should alter Bootstrap as well for the vertical lines.